[debug] [2020-04-02T23:01:06.517Z] ----------------------------------------------------------------------
[debug] [2020-04-02T23:01:06.519Z] Command:       C:\Program Files\nodejs\node.exe C:\Users\sufiy\AppData\Roaming\npm\node_modules\firebase-tools\lib\bin\firebase.js init
[debug] [2020-04-02T23:01:06.519Z] CLI Version:   8.0.1
[debug] [2020-04-02T23:01:06.520Z] Platform:      win32
[debug] [2020-04-02T23:01:06.520Z] Node Version:  v13.12.0
[debug] [2020-04-02T23:01:06.521Z] Time:          Fri Apr 03 2020 04:31:06 GMT+0530 (India Standard Time)
[debug] [2020-04-02T23:01:06.521Z] ----------------------------------------------------------------------
[debug] [2020-04-02T23:01:06.521Z] 
[debug] [2020-04-02T23:01:06.531Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2020-04-02T23:01:06.561Z] Error: Could not load the default credentials. Browse to https://cloud.google.com/docs/authentication/getting-started for more information.
    at GoogleAuth.getApplicationDefaultAsync (C:\Users\sufiy\AppData\Roaming\npm\node_modules\firebase-tools\node_modules\google-auth-library\build\src\auth\googleauth.js:160:19)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at async GoogleAuth.getClient (C:\Users\sufiy\AppData\Roaming\npm\node_modules\firebase-tools\node_modules\google-auth-library\build\src\auth\googleauth.js:502:17)
    at async GoogleAuth.getAccessToken (C:\Users\sufiy\AppData\Roaming\npm\node_modules\firebase-tools\node_modules\google-auth-library\build\src\auth\googleauth.js:524:24)
[error] 
[error] Error: An unexpected error has occurred.
[debug] [2020-04-02T23:03:02.918Z] ----------------------------------------------------------------------
[debug] [2020-04-02T23:03:02.920Z] Command:       C:\Program Files\nodejs\node.exe C:\Users\sufiy\AppData\Roaming\npm\node_modules\firebase-tools\lib\bin\firebase.js init
[debug] [2020-04-02T23:03:02.920Z] CLI Version:   8.0.1
[debug] [2020-04-02T23:03:02.920Z] Platform:      win32
[debug] [2020-04-02T23:03:02.920Z] Node Version:  v13.12.0
[debug] [2020-04-02T23:03:02.921Z] Time:          Fri Apr 03 2020 04:33:02 GMT+0530 (India Standard Time)
[debug] [2020-04-02T23:03:02.922Z] ----------------------------------------------------------------------
[debug] [2020-04-02T23:03:02.922Z] 
[debug] [2020-04-02T23:03:02.932Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2020-04-02T23:03:02.962Z] Error: Could not load the default credentials. Browse to https://cloud.google.com/docs/authentication/getting-started for more information.
    at GoogleAuth.getApplicationDefaultAsync (C:\Users\sufiy\AppData\Roaming\npm\node_modules\firebase-tools\node_modules\google-auth-library\build\src\auth\googleauth.js:160:19)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at async GoogleAuth.getClient (C:\Users\sufiy\AppData\Roaming\npm\node_modules\firebase-tools\node_modules\google-auth-library\build\src\auth\googleauth.js:502:17)
    at async GoogleAuth.getAccessToken (C:\Users\sufiy\AppData\Roaming\npm\node_modules\firebase-tools\node_modules\google-auth-library\build\src\auth\googleauth.js:524:24)
[error] 
[error] Error: An unexpected error has occurred.
[debug] [2020-04-02T23:05:28.319Z] ----------------------------------------------------------------------
[debug] [2020-04-02T23:05:28.321Z] Command:       C:\Program Files\nodejs\node.exe C:\Users\sufiy\AppData\Roaming\npm\node_modules\firebase-tools\lib\bin\firebase.js init
[debug] [2020-04-02T23:05:28.322Z] CLI Version:   8.0.1
[debug] [2020-04-02T23:05:28.322Z] Platform:      win32
[debug] [2020-04-02T23:05:28.322Z] Node Version:  v13.12.0
[debug] [2020-04-02T23:05:28.323Z] Time:          Fri Apr 03 2020 04:35:28 GMT+0530 (India Standard Time)
[debug] [2020-04-02T23:05:28.323Z] ----------------------------------------------------------------------
[debug] [2020-04-02T23:05:28.323Z] 
[debug] [2020-04-02T23:05:28.334Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2020-04-02T23:05:28.365Z] Error: Could not load the default credentials. Browse to https://cloud.google.com/docs/authentication/getting-started for more information.
    at GoogleAuth.getApplicationDefaultAsync (C:\Users\sufiy\AppData\Roaming\npm\node_modules\firebase-tools\node_modules\google-auth-library\build\src\auth\googleauth.js:160:19)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at async GoogleAuth.getClient (C:\Users\sufiy\AppData\Roaming\npm\node_modules\firebase-tools\node_modules\google-auth-library\build\src\auth\googleauth.js:502:17)
    at async GoogleAuth.getAccessToken (C:\Users\sufiy\AppData\Roaming\npm\node_modules\firebase-tools\node_modules\google-auth-library\build\src\auth\googleauth.js:524:24)
[error] 
[error] Error: An unexpected error has occurred.
